sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -50,6 +50,21 @@ namespace Azure { namespace Storage { namespace Blobs {
       const std::string& ToString() const { return m_value; }
       AZ_STORAGE_BLOBS_DLLEXPORT const static EncryptionAlgorithmType Aes256;
 
+    private:
+      std::string m_value;
```

Comment:
let's not add this atm, at least not in the public namespace. In case we need to remove it in the future

Reply:
Ok. I will remove  "ListBlobsShowOnlyType" enum.